com.cubrid.common.ui/src/com/cubrid/common/ui/query/Messages_ko_KR.properties

```diff
@@ -93,7 +93,7 @@ errEditorRunning=\uC9C8\uC758 \uD3B8\uC9D1\uAE30\uAC00 \uAD6C\uB3D9\uC911\uC785\
 tooManyRecord=\uAC80\uC0C9 \uACB0\uACFC\uAC00 {0}\uAC74\uC774 \uB118\uC2B5\uB2C8\uB2E4. \uACC4\uC18D\uD558\uC2DC\uACA0\uC2B5\uB2C8\uAE4C?
 skipOrNot=\uC9C8\uC758 \uC624\uB958\uAC00 \uC788\uC73C\uBA70 \uC624\uB958 \uB0B4\uC6A9\uC740 \uC544\uB798\uC640 \uAC19\uC2B5\uB2C8\uB2E4.\n\uC624\uB958\uB97C \uBB34\uC2DC\uD558\uACE0 \uB098\uBA38\uC9C0 \uC9C8\uC758\uB97C \uACC4\uC18D \uC2E4\uD589\uD558\uC2DC\uACA0\uC2B5\uB2C8\uAE4C?
 queryStat=Query Statistics
-querySeq=\uBC88\uC9F8 \uC9C8\uC758
+querySeq={0} \uC9C8\uC758
```

Comment:
I think this is good improvement for all users.
But spelling of the order in korean is all the same so that is not a problem in korean.
Would you please rollback this message in Messages_ko_KR.properties like before?